src/main/java/com/exactpro/th2/fix/client/Main.java

```diff
@@ -114,41 +114,45 @@ public static void main(String[] args) throws Exception {
 
         JsonMapper mapper = JsonMapper.builder().build();
         Settings settings = factory.getCustomConfiguration(Settings.class, mapper);
-
-        Map<String, String> dictionaries = new HashMap<>();
+        Path temporaryDirectory = Files.createTempDirectory("temporaryDirectory");
 
         try (InputStream rawBase64 = factory.readDictionary();
              ByteArrayInputStream byteArrayInputStream = new ByteArrayInputStream(Base64
                      .getDecoder()
                      .decode(rawBase64.readAllBytes()));
              ZipInputStream zipInputStream = new ZipInputStream(byteArrayInputStream)) {
+
             ZipEntry zipEntry;
             while ((zipEntry = zipInputStream.getNextEntry()) != null) {
                 String zipName = zipEntry.getName();
                 if (!DICTIONARY_FILE_NAME.matcher(zipName).matches()) {
```

Comment:
Why do we still have dictionary file name constraints? The only reasonable constraints now are that file should have `xml` extension and that path should be relative